tests/TestHarness/launcher.py

```diff
@@ -375,8 +375,8 @@ def init_genesis(self):
             with open(genesis_path, 'r') as f:
                 genesis = json.load(f)
         genesis['initial_key'] = self.network.nodes['bios'].keys[0].pubkey
-        genesis['initial_configuration']['max_block_cpu_usage'] = self.args.max_block_cpu_usage
-        genesis['initial_configuration']['max_transaction_cpu_usage'] = self.args.max_transaction_cpu_usage
+        genesis['initial_configuration']['max_block_cpu_usage'] = 500000 if self.args.max_block_cpu_usage is None else self.args.max_block_cpu_usage
+        genesis['initial_configuration']['max_transaction_cpu_usage'] = 475000 if self.args.max_transaction_cpu_usage is None else self.args.max_transaction_cpu_usage
```

Comment:
Instead of duplicating the default value here (magic number), I might make the setting of `genesis['initial_configuration']['max_*']` conditional on whether `self.max_* is not None`.  That way you also don't overwrite what is coming in from the loaded genesis file.

Reply:
I don't understand. It is conditional on `self.max_* is not None`.

Reply:
`if self.args.max_block_cpu_usage is not None: genesis['initial_configuration']['max_block_cpu_usage'] = self.args.max_block_cpu_usage` allows you to not have to respecify the default that we apply when initializing the genesis not from a file.  Otherwise, we are 1) duplicating the specification of a default value as a magic number 2) Possibly overwriting an intentional configuration option in a loaded `genesis.json` file.

Reply:
Done